### ISSUE TITLE
Alt text needs fixing

### ISSUE THREAD

**Issue by OriolAbril** [pymc-devs/pymc-data-umbrella#226]
I have just noticed that all the images in https://github.com/pymc-devs/pymc-data-umbrella/blob/main/2023-03_sprint/sprint_parties/contributors.md?plain=1 have "Image for contributor template" as alt text.

Following the advise in https://webaim.org/techniques/alttext/, and given all images are immediately followed by the name of the person they picture, I think it would be appropriate to use `alt=""` (similar to example 2). Another option could be using the name as alt text too (similar to example 1). Either way, I think the current content is negative and worse than having no alt text.

cc @reshamas @symeneses Once we agree on one of the two options above I think this can be labeled "good first issue"

**Comment by symeneses** [comment]
According to WebAIM, it's better to leave it empty to avoid being redundant as the name is near the image. So, I think that's the best option.
